engine/installation/linux/debian.md

```diff
@@ -287,13 +312,14 @@ installing Docker.
     Use this command to install the latest version of Docker:
 
     ```bash
-    $ sudo apt-get install docker
+    $ sudo apt-get install docker-engine
```

Comment:
When you use the "old URL" as it is the case in this section for Raspbian, there is no `docker` (there is but it is that Debian package for KDE) and there is no `docker-ce` it is still called `docker-engine` as it was previously.

My assumption is that this is transitional, in order for people with the old repository to get the updated version. The question is how long this old repo is going to be maintained by the Docker team?